src/main/java/org/rumbledb/expressions/flowr/ForClause.java

```diff
@@ -64,10 +64,7 @@ public ForClause(
         // If the sequenceType is specified, we have to "extend" its arity to *
         // because TreatIterator is wrapping the whole assignment expression,
         // meaning there is not one TreatIterator for each variable we loop over.
```

Comment:
I believe this comment should be moved to the translationVisitor where the code now lives.

Reply:
DONE